apps/web/src/routes/AuthLayoutRouter/components/ContextMenu/MenuLink/MenuLink.jsx

```diff
@@ -22,14 +36,28 @@ export default function MenuLink ({ badgeCount = null, to, children, onClick, ex
       : `https://${externalLink}`
 
     return (
-      <a href={url} target='_blank' rel='noreferrer' onClick={onClick} className={cn('MenuLink text-foreground text-sm', className, { 'opacity-100 border-selected': isCurrentLocation })}>
+      <a
+        href={url}
+        target='_blank'
+        rel='noreferrer'
+        onClick={onClick}
+        ref={isCurrentLocation ? linkRef : null}
```

Comment:
i dont think we need to check for isCurrentLocation here, its ok to add the ref to every link